tools/bake.py

```diff
@@ -367,6 +368,36 @@ def bake_pass(self, context, bake_name, bake_type, bake_pass_filter, objects, ba
                     for node in obj.active_material.node_tree.nodes:
                         if node.type == "VALUE" and node.label == "bake_" + bake_name:
                             node.outputs["Value"].default_value = 0
+        
+        
+        if context.scene.bake_optimize_solid_materials: 
```

Comment:
This also needs to be disabled if any decimation is happening

Reply:
Will do